msa/black-box-tests/src/test/java/org/thingsboard/server/msa/connectivity/MqttClientTest.java

```diff
@@ -311,7 +308,7 @@ public void clientSideRpc() throws Exception {
         // Create a new root rule chain
         RuleChainId ruleChainId = createRootRuleChainForRpcResponse();
 
-        TimeUnit.SECONDS.sleep(3);
+        TimeUnit.SECONDS.sleep(slowQueue ? 30 : 3);
```

Comment:
Maybe create a variable based on the queue type in the "before" method like QUEUE_TIMEOUT(30 or 3) ?